test/unit.js

```diff
@@ -13,7 +13,23 @@ const mocha = new Mocha({
   grep: argv.grep ? argv.grep : undefined,
 });
 
-Glob(`src/**/${argv.component ? argv.component : '*'}.spec.js`, {}, (err, files) => {
+const paths = [
+  `src/**/*${argv.component ? argv.component : ''}*.spec.js`,
+];
+
+if (!argv.component) {
+  paths.push('packages/material-ui-codemod/transforms/**/*.spec.js');
```

Comment:
As I mentioned above -- I think if we're running these tests from the main test suite that we need to have a better way of making sure all the dependencies are installed for `npm run test` without the user needing to first go to that package subfolder and running `npm install`.